cmd/agent/app/reporter/tchannel/reporter.go

```diff
@@ -57,6 +57,7 @@ type Reporter struct {
 	peerListMgr    *peerlistmgr.PeerListManager
 	batchesMetrics map[string]batchMetrics
 	logger         *zap.Logger
+	serviceName string
```

Comment:
This looks odd... Is it missing a `go fmt`?

Reply:
it's wip without formatting and tests